tests/parsing/parchmint-parser.test.js

```diff
@@ -84,5 +85,6 @@ describe('layers', () => {
             expect(l[0].validate()).toBe(true);
             expect(l[1].validate()).toBe(true);
         });
+         */
```

Comment:
Are you planning on re-introducing this test later or can it be removed?

Reply:
I was going to reintroduce it, but it will end up being pretty different after all of the parsing is done, so I've removed it for now.